python/cugraph/structure/graph.py

```diff
@@ -565,945 +277,394 @@ def from_numpy_matrix(self, np_matrix):
-
-    def view_transposed_adj_list(self):
-        """
-        Display the transposed adjacency list. Compute it if needed.
-
+        df: cudf.DataFrame or dask_cudf.DataFrame
+            A DataFrame containing external vertex identifiers that will be
+            converted into internal vertex identifiers.
+        internal_column_name: string
+            Name of column to contain the internal vertex id
+        external_column_name: string or list of strings
+            Name of the column(s) containing the external vertex ids
+        drop: (optional) bool, defaults to True
+            Drop the external columns from the returned DataFrame
+        preserve_order: (optional) bool, defaults to False
+            Preserve the order of the data frame (requires an extra sort)
         Returns
-        -------
-        offset_col : cudf.Series
-            This cudf.Series wraps a gdf_column of size V + 1 (V: number of
-            vertices).
-            The gdf column contains the offsets for the vertices in this graph.
-            Offsets are in the range [0, E] (E: number of edges).
-        index_col : cudf.Series
-            This cudf.Series wraps a gdf_column of size E (E: number of edges).
-            The gdf column contains the destination index for each edge.
-            Destination indices are in the range [0, V) (V: number of
-            vertices).
-        value_col : cudf.Series or ``None``
-            This pointer is ``None`` for unweighted graphs.
-            For weighted graphs, this cudf.Series wraps a gdf_column of size E
-            (E: number of edges).
-            The gdf column contains the weight value for each edge.
-            The expected type of the gdf_column element is floating point
-            number.
-
+        ---------
+        df : cudf.DataFrame or dask_cudf.DataFrame
+            Original DataFrame with new column containing internal vertex
+            id
         """
-        if self.distributed:
-            raise Exception("Not supported for distributed graph")
-        if self.transposedadjlist is None:
-            if self.adjlist is not None and type(self) is Graph:
-                off, ind, vals = (
-                    self.adjlist.offsets,
-                    self.adjlist.indices,
-                    self.adjlist.weights,
-                )
-            else:
-                (
-                    off,
-                    ind,
-                    vals,
-                ) = graph_primtypes_wrapper.view_transposed_adj_list(self)
-            self.transposedadjlist = self.transposedAdjList(off, ind, vals)
-
-            if self.batch_enabled:
-                self._replicate_transposed_adjlist()
-
-        return (
-            self.transposedadjlist.offsets,
-            self.transposedadjlist.indices,
-            self.transposedadjlist.weights,
+        return self.renumber_map.add_internal_vertex_id(
```

Comment:
`self.renumber_map` no longer exists in the `graph` class. Similar recommendation as above if I am not wrong

Reply:
https://github.com/rapidsai/cugraph/pull/1404#discussion_r604365353